src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Url/Base64UrlEncoder.cs

```diff
@@ -0,0 +1,442 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+using System.Text;
+using static System.Buffers.Text.Base64;
+
+namespace System.Buffers.Text
+{
+    public static partial class Base64Url
+    {
+        /// <summary>
+        /// Encodes the span of binary data into UTF-8 encoded text represented as Base64Url.
+        /// </summary>
+        /// <param name="source">The input span which contains binary data that needs to be encoded.</param>
+        /// <param name="destination">The output span which contains the result of the operation, i.e. the UTF-8 encoded text in Base64Url.</param>
+        /// <param name="bytesConsumed">When this method returns, contains the number of input bytes consumed during the operation. This can be used to slice the input for subsequent calls, if necessary. This parameter is treated as uninitialized.</param>
+        /// <param name="bytesWritten">When this method returns, contains the number of bytes written into the output span. This can be used to slice the output for subsequent calls, if necessary. This parameter is treated as uninitialized.</param>
+        /// <param name="isFinalBlock"><see langword="true"/> when the input span contains the entirety of data to encode; <see langword="false"/> when more data may follow,
+        /// such as when calling in a loop, subsequent calls with <see langword="false"/> should end with <see langword="true"/> call. The default is <see langword="true" />.</param>
+        /// <returns>One of the enumeration values that indicates the success or failure of the operation.</returns>
+        /// <remarks>The output will not be padded even if the input is not a multiple of 3.</remarks>
```

Comment:
I'd phrase the "padding is optional, so we skip it" as

```suggestion
        /// <remarks>This implementation of the base64url encoding omits the optional padding characters.</remarks>
```

That might just be a "me" thing, but I think it's more in line with the normal "voice" in our documentation.

Reply:
Sounds good